storage/migrations/22_gasoutputs_height_code.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+import "github.com/go-pg/migrations/v8"
+
+// Schema version 22 adds Height and ActorName to gas outputs table
+
+func init() {
+	up := batch(`
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN height int8;
```

Comment:
height needs to be `bigint`

Reply:
We ought to add an index, but we can't until we populate the height. We could add it in a later migration.

Reply:
fixed the type